recipe/meta.yaml

```diff
@@ -99,40 +99,40 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - stack
-    - perl
-    - coreutils
-    - binutils
-    - findutils
-    - pkg-config
-    - xz
-    - make
-    - automake
     - autoconf
-    - libtool
+    - automake
+    - binutils
     - bzip2
-    - ncurses
+    - coreutils
+    - findutils
     - libffi
+    - libtool
     - libxml2
-  host:
-    - gmp
-    - zlib
+    - make
     - ncurses
-    - libmagic
-    - sqlite
+    - perl
+    - pkg-config
+    - stack
+    - xz
+  host:
     - dbus
+    - gmp
     - libffi
     - libiconv
-    - openssh
+    - libmagic
     - lsof
+    - ncurses
+    - openssh
     - rsync
+    - sqlite
+    - zlib
   run:
-    - rsync
-    - git >=2.22
     - curl
-    - openssh
+    - git >=2.22
+    - gnupg >=2.1.1
```

Comment:
is that the minimal one required by git-annex?

all other changes to depends -- just resorted or some other changes too?

Reply:
I took the minimal gnupg version from your PR :)

There's the question of whether to use 'run' or 'run_constrained' dependency.   gnupg isn't strictly needed to run git-annex, but if it's present it has to be at least a given version.   The problem is that if not present in the conda env it might still be present in the underlying system, and be the wrong version; run_constrained only checks the conda env.   So I made it a run dependency in the standard package and a run_constrained dependency in the 'nodep' package.

Still not sure why the nodep package gets installed so much more often; the whole point of conda-forge is to minimize package conflicts by standardizing the versions of commonly used libs...

All other changes are just re-sorting.